config/semi_automated.yml

```diff
@@ -520,9 +523,9 @@ parameters:
             regex: 'MediaWiki_talk:Gadget-labelLister.js\|labelLister'
             link: MediaWiki talk:Gadget-labelLister.js
             namespaces: [ 0, 120 ]
-        Merge.js:
-            regex: '\[\[MediaWiki:Gadget-Merge\.js\|merge\.js\]\]'
-            link: MediaWiki talk:Gadget-Merge.js
+        Merge:
+            regex: '\[\[MediaWiki:Gadget-Merge\.js\|merge\.js\]\]|^Merged Item'
```

Comment:
I had to query the database to find out, but looks like the summaries for automatic merges are actually like:
```
/* wbmergeitems-to:0||Q5610520 */
```
or
```
/* wbmergeitems-from:0||Q25416796 */
```
This I'm sure is for localization purposes.

Something like `\/* wbmergeitems` should work.

Reply:
Done (except I used `\/\* wbmergeitems`)